### PR TITLE
feat: display soonaverse attributes

### PR DESCRIPTION
**Pull request by nicole-obrien**
## Summary

> Please summarize your changes, describing __what__ they are and __why__ they were made.

...

## Changelog

```
- Please write a list of changes
```

## Relevant Issues

> Please list any related issues using [development keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

...

### Platforms

> Please select any platforms where your changes have been tested.

- __Desktop__
  - [ ] MacOS
  - [ ] Linux
  - [ ] Windows
- __Mobile__
  - [ ] iOS
  - [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

...

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/views/dashboard/collectibles/views/CollectiblesDetailsView.svelte [hunk 1]
```diff
@@ -161,7 +161,7 @@
                     {/each}
                 </key-value-list>
             </nft-details>
-            {#if attributes}
+            {#if attributes && attributes.length > 0}
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/collectibles/views/CollectiblesDetailsView.svelte:164]:
```suggestion
            {#if attributes?.length > 0}
```

File: packages/desktop/views/dashboard/collectibles/views/CollectiblesDetailsView.svelte [hunk 2]
```diff
@@ -172,6 +172,31 @@
                         {/each}
                     </div>
                 </nft-attributes>
+            {:else if soonaverseAttributes}
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/collectibles/views/CollectiblesDetailsView.svelte:175]: this check isnt required

File: packages/desktop/views/dashboard/collectibles/views/CollectiblesDetailsView.svelte [hunk 3]
```diff
@@ -172,6 +172,31 @@
...
+                {#if soonaverseAttributes?.props}
+                    <nft-attributes class="flex flex-col space-y-4">
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/collectibles/views/CollectiblesDetailsView.svelte:177]: pls add a spacing between the scrollbar

**Review by MarkNerdi — CHANGES_REQUESTED** [review]
some minor changes

File: packages/desktop/views/dashboard/collectibles/views/CollectiblesDetailsView.svelte (hunk 3 above)
Comment (nicole-obrien) [reply packages/desktop/views/dashboard/collectibles/views/CollectiblesDetailsView.svelte:177]: Out of scope, and already PR for this

**Review by MarkNerdi — APPROVED** [review]
LGTM

**Pull request merged**